test/tests/Xsc/XscSystem.spec.ts

```diff
@@ -0,0 +1,26 @@
+import nock from 'nock';
+import { IJfrogClientConfig, IXscVersion } from '../../../model';
+import { JfrogClient } from '../../../src';
+import { TestUtils } from '../../TestUtils';
+
+describe('Xsc System tests', () => {
+    const clientConfig: IJfrogClientConfig = TestUtils.getJfrogClientConfig();
+    const jfrogClient: JfrogClient = new JfrogClient(clientConfig);
+
+    afterAll(() => {
+        nock.cleanAll();
+    });
+
+    describe('Version tests', () => {
+        test('Version', async () => {
```

Comment:
Not necesirily test related - didn't we want to support minimal version (1.7.x)? Or did you add it somewhere else?

It's not crucial since we're fully SaaS-Multi-Tenant, so we have full control on versions, but going forward, we might want to enable features in XSC only starting with specific version

Reply:
This is all the API available, the minimal support for versions is per usage of this library, i.e at the application itself.